### PR TITLE
[Acknowledgements] update UI to react table

### PR DESCRIPTION
**Pull request by kongtiaowang**
This pull request adding react table for Acknowledgements, and also fixing the bug which can't show more than 25 pieces of data.
screen shot 2017-11-28 at 3 26 35 pm

### CODE REVIEW

File: modules/candidate_list/test/candidate_listTest.php [hunk 1]
```diff
@@ -350,6 +350,7 @@ function testPSCIDLink()
             "document.querySelectorAll('#dynamictable > tbody > tr >".
             " td.dynamictableFrozenColumn > a')[0].click()"
         );
+        sleep(1);
```
Comment (xlecours) [inline modules/candidate_list/test/candidate_listTest.php]: 1. This is not related to the module
2. If you are using this sleep to wait for the new page to load after the click , then `safeClick()` is there especially for that

File: modules/acknowledgements/jsx/columnFormatter.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ * @param {string} cell - cell content
+ * @param {arrray} rowData - array of cell contents for a specific row
+ * @param {arrray} rowHeaders - array of headers for the table
+ * @return {*} a formated table cell for a given column
+ */
+function formatAcknowledgementsColumn(column, cell, rowData, rowHeaders) {
+  return <td>{cell}</td>;
+}
+
+window.formatAcknowledgementsColumn = formatAcknowledgementsColumn;
```
Comment (xlecours) [inline modules/acknowledgements/jsx/columnFormatter.js]: This is not necessary if you export

Comment (kongtiaowang) [reply modules/acknowledgements/jsx/columnFormatter.js]: I can only delete "export default formatAcknowledgementsColumn;".
when I delete "window.formatAcknowledgementsColumn = formatAcknowledgementsColumn;",
the react table do not show up.

File: modules/candidate_list/test/candidate_listTest.php (hunk 1 above)
Comment (kongtiaowang) [reply modules/candidate_list/test/candidate_listTest.php]: It works on PHP 7.1. Sometimes it doesn't work on PHP 7.0.
This is a react link, using normal safeclick function can't find the react element.

**Comment by kongtiaowang** [comment]
@xlecours Please review again.

File: modules/acknowledgements/jsx/columnFormatter.js (hunk 2 above)
Comment (xlecours) [reply modules/acknowledgements/jsx/columnFormatter.js]: I see. we sill fix this later when the ReactDOM.render will be removed from the templates.
